package.json

```diff
@@ -176,7 +176,7 @@
     "pump": "^3.0.0",
     "punycode": "^2.1.1",
     "qrcode-generator": "1.4.1",
-    "react": "^16.12.0",
+    "react": "16.14.0",
```

Comment:
I had to pin the version so we can dependably extract it in `.eslintrc.js`. See that file for details.

Reply:
Actually, no: 4e4f08a